packages/@aws-cdk/aws-redshift-alpha/README.md

```diff
@@ -543,3 +543,19 @@ const cluster = new Cluster(this, 'Redshift', {
 });
 cluster.addIamRole(role);
 ```
+
+## Resizing
+
+As your data warehousing needs change, it's possible to resize your Redshift cluster, but it's important to understand certain caveats.
+Most importantly, if the cluster was deployed via CDK, the cluster should be resized via CDK so it's registered in the AWS CloudFormation template.
+This ensures the `NumberOfNodes` parameter gets updated so the correct type of resize is done. There are two types of resize operations:
+
+* Elastic resize - You can add nodes to or remove nodes from the cluster. Also, you can change the node type, such as from DS2 nodes to RA3 nodes.
+AWS recommmends elastic resize as the first option, as it's a fast operation and typically completes in minutes. Elastic resize is appropriate when you:
+  * Add or reduce nodes in your existing cluster, but don't change the node type.
+  * Change the node type of your cluster.
+
+* Classic resize - Number of nodes, node type, or both, can be changed. This operation takes longer to complete, but is useful when the resize operation
+doesn't meet the criteria of an elastic resize. See [AWS Knowledge Center](https://repost.aws/knowledge-center/redshift-troubleshoot-elastic-resize) for further conditions that require classic resizing.
```

Comment:
It feels a bit like teasing to say "see this other place for the conditions". Can you summarize? Or is it by exclusion, everything that's not an elastic resize is a classic resize?

Also, do people have to do anything to their code to pick one or the other resize? Please explain if they do or do not.